src/main/java/ca/corefacility/bioinformatics/irida/service/impl/sample/SampleServiceImpl.java

```diff
@@ -801,4 +808,15 @@ public Long getSamplesCreated(Date createdDate) {
 	public List<GenericStatModel> getSamplesCreatedGrouped(Date createdDate, StatisticTimePeriod statisticTimePeriod) {
 		return sampleRepository.countSamplesCreatedGrouped(createdDate, statisticTimePeriod.getGroupByFormat());
 	}
+
+	/**
+	 * {@inheritDoc}
+	 */
+	@Transactional(readOnly = true)
+	@PreAuthorize("hasRole('ROLE_ADMIN') or hasPermission(#project, 'canReadProject')")
+	public Map<Long, Long> getCoverageForSamplesInProject(Project project, List<Long> sampleIds) {
+		return psjRepository.calculateCoverageForSamplesInProject(project, sampleIds)
+				.stream()
```

Comment:
Can we update `m` and `t` to something more meaningful so that it's clear what it is?

Reply:
Updated in cee5f10